app/src/main/java/eu/kanade/tachiyomi/ui/manga/track/TrackInfoDialog.kt

```diff
@@ -824,7 +824,11 @@ private data class TrackerRemoveScreen(
 
         fun deleteMangaFromService() {
             screenModelScope.launchNonCancellable {
-                (tracker as DeletableTracker).delete(track)
+                try {
+                    (tracker as DeletableTracker).delete(track)
+                } catch (e: Exception) {
```

Comment:
this'll also fail on network errors

Reply:
I know, since the action is to remove track so just try to catch any error on the side task of also remove entry from service.
Otherwise might need to implement:
- Check for network, if no network, no internet, failed to connect then won't allow user to remove track.
- If can connect to server then verify if the manga is exist then remove it from service, handle the return code to see if anything is wrong, maybe server got internal error which won't allow removing entry right now. With that we prevent user from remove the track.

And must implement that for all trackers. I would choose to prevent the app from crash first.